src/components/notifications/WebhookConfigModal.tsx

```diff
@@ -0,0 +1,372 @@
+                                <Tippy className="default-tt" arrow={true} trigger={"click"}
+                                    interactive={true}
+                                    placement="top" content={
+                                        <a href="https://slack.com/intl/en-gb/help/articles/115005265063-Incoming-webhooks-for-Slack" target="_blank" rel="noopener noreferrer"
+                                            style={{ color: "white", textTransform: "none" }}>
+                                            Learn how to setup slack webhooks
+                                        </a>
+                                    }>
+                                    <Help className="ml-5 dc__vertical-align-middle icon-dim-16 cursor" />
+                                </Tippy>
+                            </span>
+                            <input data-testid="add-webhook-url" className="form__input" type="text" name="app-name"
+                                value={this.state.form.webhookUrl}
+                                placeholder="Enter Incoming Webhook URL" tabIndex={2} onChange={this.handleWebhookUrlChange}
+                                onBlur={(event) => this.isValid(event, 'webhookUrl')} />
+                            <span className="form__error">
+                                {!this.state.isValid.webhookUrl
+                                    ? <><Error className="form__icon form__icon--error" />This is a required field. <br /></>
+                                    : null}
+                            </span>
+                        </label>
+                        <div className="form__row w-100-imp" >
+                            <div className="flex ml-0 dc__content-space">
+                                <span className="form__label">Headers
+                                </span>
+                                <span className="flex dc__align-end dc__content-end cb-5 fw-6 fs-13 flex right mb-4 cursor" data-testid="add-new-header-button" onClick={this.addNewHeader}>
+                                    <Add className="icon-dim-20 fcb-5" /> Add
+                                </span>
+                            </div>
+                            {this.renderHeadersList()}
+                        </div>
+                        <label className="form__row w-100-imp">
+                            <span className="form__label dc__required-field">Data to be shared through Webhook</span>
+                            <div className="dc__border pt-8 pb-8 br-4" onBlur={(event) => this.isValid(event, 'payload')}>
+                                <CodeEditor
+                                    value={this.state.form.payload}
+                                    theme="vs-alice-blue"
+                                    mode="json"
+                                    onChange={(value) => this.handleWebhookPaylodChange(value)}
```

Comment:
we ca directly write `onChange={this.handleWebhookPaylodChange}`

Reply:
done